Parsers/ASimRegistryEvent/Parsers/ASimRegistryEventMicrosoftWindowsEvent.yaml

```diff
@@ -15,7 +15,7 @@ References:
     - Title: Reference for access rights
```

Comment:
can we test filtering on this parser using ASC portal?

Reply:
yes